tests/unit_tests/gui/conftest.py

```diff
@@ -437,6 +426,15 @@ def handle_add_dialog():
 
         dialog.close()
 
-    QTimer.singleShot(1000, handle_dialog)
-    manage_tool = gui.tools["Manage cases"]
-    manage_tool.trigger()
+    with_manage_tool(gui, qtbot, handle_dialog)
+
+
+V = TypeVar("V")
+
+
+def get_child(gui, typ: Type[V], *args, waiter=None, **kwargs) -> V:
```

Comment:
Would it be more accurate to extend name to include the waiting-functionality the qtbot actually does?
It does not just get_child/return child.

Reply:
Perhaps add a wait function that uses this function?

Reply:
Something like a separate `wait_for_child` ?